ci-operator/step-registry/rosa/operator/e2e/rosa-operator-e2e-commands.sh

```
@@ -15,6 +15,12 @@ if [[ -z "${OPERATOR_NAME:-}" ]]; then
     exit 1
 fi
 
+# Export cluster ID for tests that need it (e.g., SFO checks OCM_CLUSTER_ID)
+if [[ -n "${SHARED_DIR:-}" && -f "${SHARED_DIR}/cluster-id" ]]; then
+    export OCM_CLUSTER_ID=$(cat "${SHARED_DIR}/cluster-id")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)